### PR TITLE
Add <EditableTicket />

### PR DESCRIPTION
**Pull request by ryonryon**

### CODE REVIEW

File: src/components/Column/Column.tsx [hunk 1]
```diff
@@ -7,12 +7,13 @@ import Card from "../Card";
...
 import { ColumnFooter } from "./ColumnFooter";
 import { ColumnBody, DnDColumnBody } from "./ColumnBody";
+import _EditableTicket from "../EditableTicket";
 
 export interface ColumnProps {
   column: ColumnDefinition;
   draggable?: boolean;
   onEditClick?: () => void;
-  onAddTicket?: () => void;
+  onAddTicket?: (value: string) => void;
```
Comment (kei95) [inline src/components/Column/Column.tsx:16]: Does this need to be optional?

File: src/components/EditableTicket/EditableTicket.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
+import { useEffect } from "react";
+import { CSSProperties, FormEvent, KeyboardEvent, useRef, useState } from "react";
+import styled from "styled-components";
+
+import Card from "../Card";
+
+export interface EditableTicketProps {
+  value?: string;
+  placeholder?: string;
+  onBlur?: (value: string) => void;
```
Comment (kei95) [inline src/components/EditableTicket/EditableTicket.tsx:10]: It looks like you only use `onBlur` among these three props. Do you think we'll need these properties passed by parents? Also, I think `onBlur` doesn't really need to be optional for now

**Review by kei95 — APPROVED** [review]

File: src/components/EditableTicket/EditableTicket.test.tsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    render(<EditableTicket placeholder={mockedPlaceholder} />);
+    const renderedEditableTicket = screen.getByLabelText("editable-ticket");
+
+    // Assert
+    expect(renderedEditableTicket).toHaveProperty("placeholder", mockedPlaceholder);
+  });
+
+  describe("Passed onBlur", () => {
+    test(" - focus out should fire passed function", () => {
```
Comment (kei95) [inline src/components/EditableTicket/EditableTicket.test.tsx:28]: I like the way you divide test cases👍

File: src/components/EditableTicket/EditableTicket.tsx (hunk 2 above)
Comment (ryonryon) [reply src/components/EditableTicket/EditableTicket.tsx:10]: IMHO, This component is from input component of HTML. And all on... attributes (e.g. onChange, onBlur..) are optional. so I would usually leave them as optional. But I am not sure how other component libraries are doing.

**Pull request merged**